### DISCUSSION TITLE
Don't run as admin?

### DISCUSSION THREAD

**Discussion by mitch4270** [KSP-CKAN/CKAN#4122] [General]
On Installation I get the don't run as admin message. I'm the only account on my PC so have no choice unless I make another user account with normal user rights. It's the first time I've seen this message and I've been using ckan for a while just not updated versions. I've tried using the asroot command but it did nothing either. Will this feature be staying or going as its a pain.

**Comment by phanormethtrodex** [comment]
'asroot' isn't a command, it's an arg, dude. for ckan.exe

as in '--asroot'

for ex, via cmd.exe:

> `Microsoft Windows [Version 6.1.7601]`
> `Copyright (c) 2009 Microsoft Corporation. All rights reserved.`
> 
> `C:\path\to\ckan>ckan.exe --asroot`
> `Warning: CKAN is running as an administrator user!`

and then it runs no probs. 

or create a shortcut to ckan.exe, and in the shortcut properties>Shortcut tab, Target: text box just append " --asroot"


then its just a double click and no need to fiddle with cd'ing in a cmd every time

**Comment by HebaruSan** [comment]
I think @JonnyOThan's usual advice is "Turn on UAC", the instructions for which may be found with a web search.

**Comment by DrawChilla** [comment]
I've encountered a problem where CKAN won't start. I've looked at the thread and realized that it takes a long time to launch the game through cmd every time, so I'm using a bat file.
**Code:**
_@echo off
cd /d "C:\Users\"ur username"\Desktop"
ckan.exe --asroot_

**Reply by JonnyOThan** [reply]
This is not a good idea, but it can work as a last resort.

If CKAN is warning you that it's running as admin, then it means that you are running EVERY program as admin.  This is really terrible for your computer, security-wise.

Usually, people in this situation just need to turn on UAC in their windows settings.

More rarely, even that doesn't work.  As far as I've been able to learn, there is a hidden "super admin" flag that some users have, usually when they originated on windows 7 and earlier.  There's also one super admin account by default on windows 10 and 11, but it's disabled by default and you shouldn't be using this for daily use.

There doesn't seem to be any way to remove the super-admin flag from a user account that I've been able to find.  So if you're using one of those accounts, your choices are:

1. Create a new, normal user account (can even be admin, as long as you use UAC) and use that for daily stuff.
2. Launch CKAN with `--asroot` and ignore the problem

**Comment by wolfycom4** [comment]
i scanned it for viruses because i got warned and found "Not-a-virus.ZipBomb"

**Comment by JonnyOThan** [comment]
@wolfycom4 seems like you downloaded the source code repository and scanned it. Which is not what you should be doing if you want to install ckan.

yes, the unit tests for ckan contain a zip bomb file, because we make sure that ckan does not download and install them on users’ machines.
